src/pcre2.h.generic

```diff
@@ -912,6 +923,7 @@ pcre2_compile are called by application code. */
 #define pcre2_set_newline                     PCRE2_SUFFIX(pcre2_set_newline_)
 #define pcre2_set_parens_nest_limit           PCRE2_SUFFIX(pcre2_set_parens_nest_limit_)
 #define pcre2_set_offset_limit                PCRE2_SUFFIX(pcre2_set_offset_limit_)
+#define pcre2_set_optimize                    PCRE2_SUFFIX(pcre2_set_optimize_)
```

Comment:
Where is the definition of the function?

Reply:
In pcre2_context.c, same as all the other pcre2_set_* functions.

Reply:
should be also exported after alexdowad/pcre2#6